docs/cookbook.rst

```diff
@@ -284,8 +284,8 @@ resulting code will look something like::
         def obj_create(self, bundle, **kwargs):
             return super(EnvironmentResource, self).obj_create(bundle, user=bundle.request.user)
 
-        def apply_authorization_limits(self, request, object_list):
-            return object_list.filter(user=request.user)
+        def get_object_list(self, request):
```

Comment:
We should instead recommend overriding authorized_read_list, authorized_read_detail, etc.